feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -3,7 +3,6 @@ export default class DataAccount {
         this.importingResults = [];
         this.importedFileNames = [];
         this.name = "";
-        this.preferredLanguage = [];
```

Comment:
Is this really not needed in the importers?

Reply:
No, it's not used.